.gitignore

```diff
@@ -44,3 +44,8 @@ secret
 ###< symfony/framework-bundle ###
 /bin/.phpunit/
 /doc/en/_build/
+
+src/Centreon*
```

Comment:
src/Centreon should not be ignored

Reply:
there are some other directories beginning by Centreon : https://github.com/centreon/centreon/tree/master/src